pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -582,10 +581,8 @@ func translateCORSPolicy(in *networking.CorsPolicy) *route.CorsPolicy {
 	}
 
 	out := route.CorsPolicy{
-		AllowOrigin: in.AllowOrigin,
-		EnabledSpecifier: &route.CorsPolicy_Enabled{
-			Enabled: proto.BoolTrue,
-		},
+		AllowOrigin:      in.AllowOrigin,
+		EnabledSpecifier: &route.CorsPolicy_FilterEnabled{},
```

Comment:
Defaults to 100/HUNDRED which is equivalent to TRUE

Reply:
I think the default is **0**/HUNDRED instead of 100/HUNDERED (numerator default is 0, and denominator default is HUNDRED)

```
message FractionalPercent {
  // Specifies the numerator. Defaults to 0.

  uint32 numerator = 1;

  DenominatorType denominator = 2 [(validate.rules).enum.defined_only = true];
}
```

Reply:
Translateroute does not pass down model.Proxy, so we don’t have access to more metadata.
We should just pass down node metadata to these functions and generate old and new config.

It may affect caching though.

Reply:
@silentdai There is a clear note above this field:
``` 
 //   This field defaults to 100/:ref:`HUNDRED
 //   <envoy_api_enum_type.FractionalPercent.DenominatorType>`.
```
It is probably against the proto3 spec, but it is what it is.

@mandarjog how does caching work here? is it even possible to pass per-proxy identifier?

Reply:
@kyessenov I think both `100` and `HUNDRED` refer to `denominator`. No `numerator` is addressed in the comment.
What I am saying is the `numerator` is default 0.